filebeat/input/log/config.go

```diff
@@ -104,10 +104,12 @@ type config struct {
 
 	// Hidden on purpose, used by the docker input:
 	DockerJSON *struct {
-		Stream   string `config:"stream"`
-		Partial  bool   `config:"partial"`
-		ForceCRI bool   `config:"force_cri_logs"`
-		CRIFlags bool   `config:"cri_flags"`
+		Stream            string `config:"stream"`
+		Partial           bool   `config:"partial"`
+		ForceCRI          bool   `config:"force_cri_logs"`
+		CRIFlags          bool   `config:"cri_flags"`
+		KeepOriginal      bool   `config:"keep_original_message"`
```

Comment:
What about just call it `keep_original`. Got inspired as you call the variable like this. I think adding `message` could even be confusing as it's not the original message (of the json) that we keep but the original of the full json blob.